cogs/vxtwitconverter/commandsCore.py

```diff
@@ -0,0 +1,13 @@
+from redbot.core.commands.context import Context
+
+from .core import Core
+from .constants import KEY_ENABLED
+
+
+class CommandsCore(Core):
+    async def cmdToggle(self, ctx: Context):
+        enabled = not await self.config.guild(ctx.guild).get_attr(KEY_ENABLED)()
+        await self.config.guild(ctx.guild).get_attr(KEY_ENABLED).set(enabled)
```

Comment:
Can extract to variable.

```suggestion
        enabledCfg = self.config.guild(ctx.guild).get_attr(KEY_ENABLED)
        enabled = not await enabledCfg()
        await enabledCfg.set(enabled)
```